pkgs/applications/audio/guitarix/default.nix

```diff
@@ -42,30 +41,15 @@ let
   inherit (lib) optional;
 in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "guitarix";
-  version = "0.44.1";
+  version = "0.46.0";
 
   src = fetchurl {
```

Comment:
Use `fetchFromGitHub` instead. This will still fetch a tarball / zip of the desired repo tag, but is easier to read and can be updated automatically when a new release tag is pushed.